### ISSUE TITLE
Thanks!

### ISSUE THREAD

**Issue by nongdenchet** [ntony263/TodoApp#1]
#### Good stuffs
- Using `ListView` and `ArrayAdapter` to display data
- Add listener to remove item
- Nice use of `Toast`
- Nice use of `onActivityResult`
- Able to store data in file
#### Suggestions
- We have received many other good submission. So I really love to see you implement some of the `optional` requirements

**Comment by ntony263** [comment]
Thank you for your comment
I would like to ask you one question, the deadline for submission is midnight 29th Sep that means at the end of the day 29th Sep? (or at the start of the day?)

**Comment by nongdenchet** [comment]
It will be the end of the day @ntony263

**Comment by ntony263** [comment]
@nongdenchet I have improved my app. Could you please have a look at it
Thank you

**Comment by nongdenchet** [comment]
Sorry for late response I will give review as soon as possible

**Comment by nongdenchet** [comment]
Hey do you use gradle for building the project, I could not build your source because missing of `gradle` file in root directory, could u check

**Comment by ntony263** [comment]
I don't use anything speacial in this 2nd version, I just work on the same file with first version and push it to github. Could you give me the error log? @nongdenchet

**Comment by ntony263** [comment]
I have updated my todolist file. I still dont know where the problem is. But it runs without a problem on my PC. Could you please have a look at it again

**Comment by nongdenchet** [comment]
I will give review as soon as possible my friend

**Comment by nongdenchet** [comment]
Hi can you check your git again, I only see the `app` directory from your github. Your repo should look like [this](https://www.dropbox.com/s/dyqsstzkw559mon/Screenshot%202016-10-01%2019.51.09.png?dl=0). Would be nice if you could commit your gradle config file

**Comment by ntony263** [comment]
Thank you for yout help. I will try to solve that problem

**Comment by nongdenchet** [comment]
Yes I will wait for you. I need your gradle file to build the project

**Comment by ntony263** [comment]
![gradle](https://cloud.githubusercontent.com/assets/20414414/19015036/4d22928a-8826-11e6-8eb0-001583f351a8.jpg) My repo looks similar to yours, but my gradle file on github is still version 1 (not change). I am investigating it now

**Comment by ntony263** [comment]
@nongdenchet Maybe I found the problem, could you please take a look at my project

**Comment by nongdenchet** [comment]
Hi I have successfully run your app
You have improve your application but we have many good submission here. If you could do a little more it would be nice:
1. Could you improve the `Priority` using `Spinner`
2. Could you improve due date using `DateTimePicker`
3. If you can try to improve the UI

How to implement is mentioned in the tutorial [here](https://guides.codepath.com/android/Working-with-Input-Views)

**Comment by ntony263** [comment]
Hi thank you for your comment. I am working on my project to improve it. Maybe today, I will submit the enhanced version

**Comment by nongdenchet** [comment]
I really love your fast response and commitment, really want you to join the class, just a little bit more effort man

**Comment by ntony263** [comment]
Hi @nongdenchet I have updated my app according to your comments
Could you please have a look at it
Thank you very much

**Comment by nongdenchet** [comment]
Hi nice to see your new stuffs. Thank you for pairing with me. I think I will give you a slot. Love your commiment

**Comment by ntony263** [comment]
Thank you very much.  I am looking forward to learning a lot more from you and from CoderSchool
